src/hex.rs

```diff
@@ -102,8 +102,11 @@ pub struct HexIterator<'a> {
 }
 
 impl<'a> HexIterator<'a> {
-    /// Constructs a new `HexIterator` from a string slice. If the string is of
-    /// odd length it returns an error.
+    /// Constructs a new `HexIterator` from a string slice.
+    ///
+    ///  # Errors
```

Comment:
In a9ed7f8294c480ee42771d5b0a296b8f84806f3f:

There is an extra space before the `#`

Reply:
Cheers, I'd love to know how to lint the rustdocs :)